### PR TITLE
Add notice about removing tfg_support

### PR DESCRIPTION
**Pull request by swebb**
Yup.

### CODE REVIEW

**Comment by osiro** [comment]
:+1: 

Suggestion:

Add the following to `tfg_support.gemspec`

```ruby
spec.post_install_message = "Gem is deprecated and no longer maintained."
```

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# NOTICE
+
+DO NOT USE.
+
+THIS GEM SHOULD NOT BE USED. IT QUICKLY BECAME A DUMPING GROUND FOR HACKY METAPROGRAMING SOLLUTIONS. IT MODIFIED MANY CORE RUBY OBJECTS IN FUNDUMENTAL WAYS, BREAKING OTHER RUBY CODE IN UNEXPECTED WAYS. IT IS ONLY KEPT HERE TO ALLOW EXISTING PROJECTS TO CONTINUE TO WORK. ATTEMPT TO REMOVE IT FROM PROJECTS WHERE POSSIBLE.
```
Comment (pzi) [inline README.md]: typo `Metaprogramming`

**Comment by pzi** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (levibuzolic) [inline README.md]: Geez @swebb spell bettererer. :smile: 

<del>SOLLUTIONS</del> => SOLUTIONS
<del>FUNDUMENTAL</del> => FUNDAMENTAL

Comment (swebb) [inline README.md]: I got hired for by ruby skills. Not my English skills :P

**Pull request merged**